src/pages/Login.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import '../styles/Login.css';
+
+import { FaUser } from "react-icons/fa";
+import { TbEyeClosed, TbEyeUp } from "react-icons/tb";
+import logo from '../assets/logo.png';
+
+import { jwtDecode } from 'jwt-decode';
+
+
+const Login = () => {
+    const navigate = useNavigate();
+    const [username, setUsername] = useState('');
+    const [password, setPassword] = useState('');
+    const [showPassword, setShowPassword] = useState(false);
+    const [errorMessage, setErrorMessage] = useState('');
+
+    const handleLogin = async (e) => {
+        e.preventDefault();
+        setErrorMessage('');
+
+        const userData = {
+            username: username,
+            password: password
+        };
+
+        try {
+            const response = await axios.post('http://localhost:8080/user/login', userData);
```

Comment:
the url is for dev. make a configuration that can be used for both dev and/or production